tools/ui-components/src/form-group/form-group.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useMemo } from 'react';
+import { FormContext } from '../form-context';
+import { FormGroupProps } from './types';
+
+let variantClass = '';
+
+const hasSuccess = `
+[&_span:has[data-testid='help-block']]: color-var(--highlight-color)
+[&_label]: color-var(--highlight-color)
+[&_input]: bordercolor-var(--quaternary-background) shadow-inner-small
+[&_input:focus]: bordercolor-var(--tertiary-color) shadow-inner-success-focus
```

Comment:
I have no idea how to change the color based on light and dark mode in theme, using prefers color scheme in tailwind doesn't cut it, because we aren't using color scheme to set the dark and light mode

https://tailwindcss.com/docs/hover-focus-and-other-states#prefers-color-scheme